pkg/ebpf/encoding/format.go

```diff
@@ -7,12 +7,26 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/process/util"
 )
 
+// AddrCache is responsible for interning util.Address strings
+type AddrCache map[util.Address]string
```

Comment:
Do we ever reset this?

Reply:
As it is it doesn't grow unbounded (it's not used globally but within a single `Marshal` function call)